mentors/README.md

```diff
@@ -1,78 +1,127 @@
+* Be available to answer questions when candidates ask questions about your project idea
 
-The Linux Foundation also offers a free short course for Beginners which can help you get familiar with the fundamentals of Open Source and started on your contributor journey.
+### Reviewing Mentee Applications and Proposals
 
+To make the mentoring experience pleasant for you and for your mentee:
+* Do not accept "OK" applications and proposals. Look for enthusiastic candidates and high quality proposals.
+* Do not accept more mentees than you can handle. If you are not sure how many mentees you can handle, ask the program administrators.
+* Set your expectations right in the project idea. You would want to avoid to be in the situation where you have a mismatch of expectations with your mentee.
+* Provide feedback to the candidates and help them improve their proposals. This will help you and your mentee in the long run.
 
+### Mentoring Mentees
 
-## FAQs: Returning Mentees - Full List
+To make the most of your mentoring experience, we recommend the following best practices:
+* Be available for the mentee! This is the most important thing you can do for your mentee and it will be the most rewarding thing for you.
+* Introduce your mentee to the community and help them get familiar with the community processes.
+* Encourage your mentee to interact with the community, and not just with you. While you will be the main responsible person for the mentee, a mentee interacting with the community will help you share the load.
+* Ask your mentee to submit their work early and often, and also ask them to look for community feedback.
+* Look for opportunities to retain the mentee in the community after the program ends. This can be done by helping them find a new project to work on.
+* Share your mentoring experience with your community and the wider open source community by writing a blog post together, giving a talk together or preparing a demo together. This will help you and your community to attract more mentees in the future.
 
-1. I've graduated a mentorship program. What's next?
-1. How do I become a Mentor for a project?
-1. I'm having trouble finding a job now I've graduated. What steps should I take?
-1. I didn't finish a program in the past. Can I still take part in mentorship opportunities?
-1. Can I apply for the LFX Mentorship again after successfully completing a Semester?
-1. I've submitted my application or requested information and haven't heard back by the expected timeframe. How can I follow up?
+## Support
 
+Mentors have a direct line to us!
 
+If you have any questions, please reach out to the CNCF mentoring group on [CNCF Slack](https://cloud-native.slack.com/archives/CGPK98JNQ) or on [CNCF Mentoring repository discussions](https://github.com/cncf/mentoring/discussions). We are happy to help you mentor your mentees!
 
+## Mentor Resources
```

Comment:
Need to link https://docs.linuxfoundation.org/lfx/mentorship/mentor-guide/getting-started/what-makes-a-good-mentor as a generic guide regardless of the program